app/main/forms.py

```diff
@@ -0,0 +1,85 @@
+from flask.ext.wtf import Form
+from wtforms.fields import StringField, SubmitField, IntegerField, TextAreaField, HiddenField, \
+    DateField, FileField
+from ..models import Agency
+from wtforms.ext.sqlalchemy.fields import QuerySelectField
+from wtforms.validators import (
+    InputRequired,
+    Length,
+    Optional,
+    NumberRange,
+    URL
+)
+from app.custom_validators import StrippedLength
+from .. import db
+import datetime as datetime
+
+
+class IncidentReportForm(Form):
+    vehicle_id = StringField('Vehicle ID', validators=[
+        InputRequired('Vehicle ID is required.'),
+        StrippedLength(
+            min_length=2,
+            max_length=15,
+            message='Vehicle ID must be between 2 to 15 characters after '
+                    'removing all non-alphanumeric characters.'
+        ),
+    ])
+
+    license_plate = StringField('License Plate Number', validators=[
+        Optional(),
+        StrippedLength(
+            min_length=4,
+            max_length=8,
+            message='License plate must be between 4 to 8 characters after '
+                    'removing all non-alphanumeric characters.'
+        )
+    ])
+
+    bus_number = IntegerField('Bus Number', validators=[
+        Optional()
+    ])
+
+    led_screen_number = IntegerField('LED Screen Number', validators=[
+        Optional()
+    ])
+
+    latitude = HiddenField('Latitude')
+    longitude = HiddenField('Longitude')
+    location = StringField('Address')
+    today = datetime.date.today()
+    date = DateField('Date', default=today,
+                     validators=[Optional()])
+
+    # TODO - add support for h:m:s format
+    duration = IntegerField('Idling Duration (minutes)', validators=[
```

Comment:
This is out of date. I think recent changes weren't incorporated because all changes have been to `reports/forms.py`